Apps/Playground/App.tsx

```diff
@@ -50,6 +50,15 @@ const EngineScreen: FunctionComponent<ViewProps> = (props: ViewProps) => {
               }
             }
           })
+        } else if (device.deviceType === DeviceType.Mouse) {
```

Comment:
Based on https://github.com/BabylonJS/BabylonNative/pull/583#discussion_r564143476, what this code should do is check for a pre-existing mouse device and if it exists, register for `onInputChangedObservable` on that device. When we see the mouse move, just poll the state of the left mouse button and see if it is down and only if so do the object rotation. This is how the code would be written when running in the browser. (Also it would be good to factor out the actual `rootNode.rotate(...)` into a local const lambda that can be called in both the touch and mouse cases so it's not repeated).

Reply:
this should now be fixed in the component. i'll follow the suggested refactor as well.